tests/SMS77Test.php

```diff
@@ -0,0 +1,50 @@
+<?php
```

Comment:
I know a lot of people have different opinions on testing API calls.  For this case, I want to avoid it.